_docs_integrate/13-connector-error-codes.md

```diff
@@ -0,0 +1,79 @@
+---
+title: ErrorCodes
+permalink: /integrate/errorCodes
+---
+
+Many errors are about false user input. If you cannot regognize a connection between your interactions and the error or you think the error is incorrect please report it in the [Enmeshed Issue Tracker](https://github.com/nmshd/feedback/issues).
+
+| ErrorCode                                                                                                             | Description                                                                                                                                                               |
+| --------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
Ich würde hier gerne noch eine Category-Spalte (z.B. Connector API, Runtime (Invalid Input), Runtime (Invalid State), Runtime (Internal)) und eine "How-to-fix"-Spalte (z.B. "Check your JSON payload for invisible characters or wrong serializations") sehen.